Code-the-Dream-School/dd-prac-team1-front
# Filtering recipes PRA-110

## Filtering recipes due to the existing categories

## Requirements

The user can filter the recipes according to the existing categories

## Notes

None

## Test Steps

When the page  /saved-recipes is rendered, you can see all saved recipes.
When you click on the category, you can see the recipes with this category.
When you click on "all recipes", you can see all recipes again.

## UI/UX Outcome

<img width="1372" alt="Screen Shot 2023-08-07 at 13 38 33" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/114008959/1e07dbcf-5b74-4ba9-b78c-af59d7ac95ab">

<img width="1331" alt="Screen Shot 2023-08-07 at 13 38 46" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/114008959/219a0c05-b41d-48cf-831f-2c88c8cdcba4">

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed
